test/cql-pytest/test_bloom_filter.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+# Test inserts `N` rows into table, flushes it 
+# and tries to read `M` non-existing keys.
+# Then bloom filter's false-positive ratio is checked.
+
+import pytest
+import random
+import nodetool
+from util import new_test_table
+
+def test_bloom_filter(cql, test_keyspace):
+    N = 10000 # Number of writes
+    M = 5000  # Number of reads
+    fp_chance = 0.1
```

Comment:
I wonder if you can't just drop a zero off N and M and still get a good enough test. How close do we need to get to fp_chance to say that this a success?

The smallest you can get these numbers and still get a reasonable test, the better.

Reply:
I've dropped the `N` and `M` to `N=500`, `M=2000` and the test takes `~0.8s`. Also, I've changed asserting range, some explanation is [here](https://github.com/scylladb/scylla/pull/10186#discussion_r826196138).

Reply:
Good.